src/NumericalAlgorithms/Spectral/FiniteDifference.cpp

```diff
@@ -25,6 +25,21 @@ std::pair<DataVector, DataVector> compute_collocation_points_and_weights<
   return std::make_pair(std::move(x), std::move(w));
 }
 
+template <>
+std::pair<DataVector, DataVector> compute_collocation_points_and_weights<
+    Basis::FiniteDifference, Quadrature::FaceCentered>(
+    const size_t num_points) noexcept {
+  DataVector x(num_points);
```

Comment:
[very nitpick] these declarations are pretty close to a 'vexing parse' problem, so might be better constructed with braces.